ci/auto/dash-bootstrap.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -eo pipefail
+
+usage() {
+
+    cat <<EOF
+Usage:
+	./scripts/$0 <dashboard base url>
+
+This script should be run from the repo root as it expects files (data/{org,api}.json) relative to it.
+The base URL for the compose env in this repo is usually http://localhost:3000. A trailing slash is not required.
+All authentication and parameters are hardcoded except for those which cannot 
+EOF
+}
+
+curlf() {
+    curl --header 'Content-Type:application/json' -s --show-error "$@"
+}
+
+if [ -z $1 ]; then
+    usage
+    exit 1
+fi
+
+if [[ "$(uname)" == "Darwin" ]]; then
+  echo "Running on macOS"
+	sed_exp="-i ''"
+else
+	sed_exp="-i"
+fi
+
+
+db_base=${1:-"http://localhost:3000"}
+
+org_json='{
+	"owner_name": "System Test Org",
+	"owner_slug": "st",
+	"cname_enabled": false,
+	"hybrid_enabled": true,
+	"cname": ""
+}'
+
+# 1. Create org using admin authentication
+orgid=$(curlf --header "admin-auth: 12345" \
+	      --data "$org_json" \
+	      ${db_base}/admin/organisations | jq -r '.Meta')
+
+echo "TYK_GW_SLAVEOPTIONS_RPCKEY=${orgid}"
+sed ${sed_exp} "s/TYK_GW_SLAVEOPTIONS_RPCKEY=.*/TYK_GW_SLAVEOPTIONS_RPCKEY=${orgid}/g" *.env
```

Comment:
This variable is required for GW SLAVES to connect with MDCB. Is propagated to all env files in the root folder using SED. Not a very elegant approach